lib/ouija_tests.ml

```diff
@@ -1,74 +1,96 @@
-open OUnit2
+open Core.Std
+open Async.Std
+open Sentinel.Std
 
-let rest _ =
+let check exp op = fun _ -> return (exp = op ())
```

Comment:
Sentinel.Basic is certainly enough. I just went for Async cause I see no reason why these tests should be run sequentially. The downside of my decision is adding an extra build dependency (Async is not needed by the Ouija code).
Just tell me if you'd rather seeing theses tests updated to Sentinel.Basic, it's a 5 minute change.